db/migrate/20210721064506_add_mentoring_stats_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddMentoringStatsToUser < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :num_solutions_mentored, :integer, limit: 3, null: false, default: 0
```

Comment:
This allows for 8388607 solutions to be mentored before overflowing, which I don't think we'll reach anytime soon. 32767 seemed like a too low limit.